prov/sm2/src/sm2_coordination.c

```diff
@@ -192,7 +192,7 @@ ssize_t sm2_file_open_or_create(struct sm2_mmap *map_shared)
 {
 	pthread_mutexattr_t att;
 	struct sm2_mmap map_ours;
-	char template[FI_NAME_MAX];
+	char template[sizeof(SM2_COORDINATION_DIR) + 32];
```

Comment:
Not in love with the magic number, but its not the end of the world for me.

Reply:
it's really: `sizeof(SM2_COORDINATION_DIR) + sizeof("/fi_sm2_pid1234567_XXXXXX") + 1` Which IIRC was something like 27.

Reply:
addressed.